classes/TaskRunner/Miscellaneous/UpdateConfig.php

```diff
@@ -95,6 +108,40 @@ public function run()
         }
     }
 
+    /**
+     * @param array $parameters
+     */
+    public function inputCLIParameters($parameters)
```

Comment:
```suggestion
    public function inputCLIParameters($parameters): array
```

Reply:
I am not sure 🤔  this code must remain php5.6-compliant if I am not wrong. Is this style compliant with php5.6 ?

Reply:
Can't use PHP 7 features in the autoupgrade module